storybookjs/storybook
# Bug: ensure src of iframe for a ref stays the same, unless the version changes

Closes https://github.com/storybookjs/storybook/issues/21499

## What I did

I ensured that when rendering iframes we don't change the src attribute of the iframe unless the base-url changes (the version changed)

## How to test

1. Add a ref
2. Start and load storybook
3. Go to the composed storybook
4. Switch between stories

Expect no loaders/spinner and no refreshes
